Tasks/RunDistributedTests/RunDistributedTests.ps1

```diff
@@ -55,6 +57,7 @@ Write-Verbose "UnregisterTestAgent script Path  = $unRegisterTestAgentLocation"
 Write-Verbose "Calling Invoke-RunDistributedTests"
 
 $checkTestAgentCompatScriptLocationMemberExists  = CmdletHasMember "CheckTestAgentCompatScriptLocation"
+$checkCustomSlicingEnabledMemberExists  = CmdletHasMember "CustomSlicingEnabled"
 $taskContextMemberExists  = CmdletHasMember "TaskContext"
```

Comment:
too many if statement. ideal way to handle would have been to construct the parameter string and then calling the cmdlet